patches/nginx-1.13.6-resolver_conf_parsing.patch

```diff
@@ -0,0 +1,192 @@
+diff --git a/src/core/ngx_resolver.c b/src/core/ngx_resolver.c
+index cd55520c..06d1c461 100644
+--- a/src/core/ngx_resolver.c
++++ b/src/core/ngx_resolver.c
+@@ -15,6 +15,9 @@
+ #define NGX_RESOLVER_TCP_RSIZE  (2 + 65535)
+ #define NGX_RESOLVER_TCP_WSIZE  8192
+ 
++#define NGX_RESOLVER_FILE_BUF_SIZE  2048
```

Comment:
If we follow the approach taken in #247 (taken from other parts of the nginx codebase), we can easily parse files that exceed this size, if desired.

https://github.com/openresty/openresty/pull/247/files#diff-1f78491f86d1e1677eca130eb1799f68R198

Reply:
I think 2KB is more than enough for all non-abusing users. It's never a good idea to read the whole file as there is a possibility of the filesystem being broken or /etc/resolv.conf being symlinked to `/dev/zero`.

My observations is that `resolv.conf` will rarely exceed 500 bytes even with crazy comments and so on, so making it larger than 2KB seems to be unnecessarily large.